### PR TITLE
fix php8 arginfo

### PR DESCRIPTION
**Pull request by m6w6**
A few fixes for stub/arginfo to match the reality, which makes the test suite complete successfully on PHP-8.0

### CODE REVIEW

**Comment by sodabrew** [comment]
@remicollet Please review as you added the most recent iteration of these stub files.

File: php_memcached.stub.php [hunk 1]
```diff
@@ -35,16 +35,16 @@ public function cas(string $cas_token, string $key, mixed $value, int $expiratio
...
 	public function add(string $key, mixed $value, int $expiration=0): bool {}
 	public function addByKey(string $server_key, string $key, mixed $value, int $expiration=0): bool {}
-	public function append(string $key, string $value): bool {}
-	public function appendByKey(string $server_key, string $key, string $value): bool {}
-	public function prepend(string $key, string $value): bool {}
-	public function prependByKey(string $server_key, string $key, string $value): bool {}
+	public function append(string $key, string $value): ?bool {}
+	public function appendByKey(string $server_key, string $key, string $value): ?bool {}
+	public function prepend(string $key, string $value): ?bool {}
+	public function prependByKey(string $server_key, string $key, string $value): ?bool {}
```
Comment (remicollet) [inline php_memcached.stub.php:41]: From `php_memc_store_impl` these method return NULL on error (cannot append/prepend with compression turned on) so another way is to raise an exception (PHP 8 only) in such case, as part of the "promote error to exception" global change.

**Review by remicollet — COMMENTED** [review]
LGTM

**Comment by sodabrew** [comment]
Thanks! Merging now.

**Comment by m6w6** [comment]
Awesome, thank you!

**Pull request merged**